examples/c/sockfilter.c

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2022 Jacky Yin */
+#include <arpa/inet.h>
+#include <assert.h>
+#include <bpf/libbpf.h>
+#include <linux/if_packet.h>
+#include <linux/if_ether.h>
+#include <linux/in.h>
+#include <net/if.h>
+#include <stdio.h>
+#include <sys/resource.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include "sockfilter.skel.h"
+
+struct so_event {
```

Comment:
this struct is used in BPF part and in user-space part, let's extract it into a common header, just like bootstrap.h does?

Reply:
It's a good idea.
I added a new socketfilter.h file.